fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/SavingsAccountTransactionType.java

```diff
@@ -124,6 +127,20 @@ public static SavingsAccountTransactionType fromInt(final Integer transactionTyp
         return savingsAccountTransactionType;
     }
 
+    public static SavingsAccountTransactionType fromString(String transactionType) {
+        for (SavingsAccountTransactionType savingsAccountTransactionType : SavingsAccountTransactionType.values()) {
```

Comment:
I really really don't like this splitting logic. Why can't we solve it differently which doesn't involve string magic? this solution is super fragile.